firefox-ios/firefox-ios-tests/Tests/ClientTests/TranslationsTests/TranslationsEngineTests.swift

```diff
@@ -0,0 +1,80 @@
+// This Source Code Form is subject to the terms of the Mozilla Public
+// License, v. 2.0. If a copy of the MPL was not distributed with this
+// file, You can obtain one at http://mozilla.org/MPL/2.0/
+
+import XCTest
+import WebKit
+@testable import Client
+
+@MainActor
+final class TranslationsEngineTests: XCTestCase {
+    func test_bridgeTo_reusesBridgeForSameWebView() {
+        let subject = createSubject()
+        let pageWebView = WKWebView()
+
+        let firstBridge = subject.bridge(to: pageWebView)
+        let secondBridge = subject.bridge(to: pageWebView)
+
+        XCTAssertTrue(firstBridge === secondBridge)
```

Comment:
If we are using this route, can we add a message so it can give us more details if this were to fail in our pipeline as opposed to just returned true or false